DNN Platform/Library/Entities/Tabs/TabController.cs

```diff
@@ -2580,6 +2622,13 @@ public static void CopyPermissionsToChildren(TabInfo parentTab, TabPermissionCol
                             listTabs.Add(tab);
                         }
                     }
+
+                    // HasChildren should be true in case there is at least one not deleted child
+                    tab.HasChildren &=
+                        includeDeletedChildren || (

```

Comment:
what if `GetTabsByParent(tab.TabID, tab.PortalID)` returns empty list, shouldn't `tab.HasChildren` be false at that time? I mean like in the following:

```
tab.HasChildren &= (
        includeDeletedChildren && GetTabsByParent(tab.TabID, tab.PortalID).Any()
    ) || (
        !includeDeletedChildren && GetTabsByParent(tab.TabID, tab.PortalID).Any(a => !a.IsDeleted));
```

or more cleanly:
`GetTabsByParent(tab.TabID, tab.PortalID).Any(a => includeDeletedChildren || !a.IsDeleted));`

Reply:
If `GetTabsByParent` is empty, I would expect that `tab.HasChildren` is already `false`, so the rest of the conditions aren't considered.  I believe it is fine as it is, but thanks for the review and double check 👍